docs/conveyor/introduction.md

```diff
@@ -0,0 +1,2 @@
+# Introduction
+Automata Conveyor is a service that ingests and outputs transactions in a determined order, which create a front-running-free zone that removes the chaos of transaction reordering.
```

Comment:
Automata Conveyor is an anti-front-running service that allows users to trade ERC20 tokens in a deterministic order via Geode-enabled trusted relayers. To use this service, users would wrap their ERC20 tokens into gTokens to enjoy the benefit of gasless trading.